prometheus/prometheus.go

```diff
@@ -71,3 +71,38 @@ func (c *Prometheus) queryInt64(ctx context.Context, query string) (int64, error
 	}
 	return int64(vec[0].Value), nil
 }
+
+type AICapacity struct {
+	TotalContainers int64 `json:"totalContainers"`
+	InUseContainers int64 `json:"inUseContainers"`
+	FreeContainers  int64 `json:"freeContainers"`
+}
+
+func (c *Prometheus) QueryAICapacity(ctx context.Context, region, nodeID string) (AICapacity, error) {
+	regionFilter, nodeIDFilter := "", ""
+	if region != "" {
+		regionFilter = fmt.Sprintf(`, region="%s"`, region)
+	}
+	if nodeID != "" {
+		nodeIDFilter = fmt.Sprintf(`, node_id="%s"`, nodeID)
+	}
+	query := fmt.Sprintf(`sum(livepeer_ai_container_free{job="orchestrator"%s%s})`, regionFilter, nodeIDFilter)
+
+	free, err := c.queryInt64(ctx, query)
+	if err != nil {
+		return AICapacity{}, err
+	}
+
+	query = fmt.Sprintf(`sum(livepeer_ai_container_in_use{job="orchestrator"%s%s})`, regionFilter, nodeIDFilter)
+
+	inUse, err := c.queryInt64(ctx, query)
+	if err != nil {
+		return AICapacity{}, err
+	}
+
+	return AICapacity{
+		TotalContainers: free + inUse,
+		InUseContainers: inUse,
+		FreeContainers:  free,
+	}, nil
```

Comment:
Yeah, as per my comment in the [other PR](https://github.com/livepeer/go-livepeer/pull/3537/files#r2059208229), this is not the complete view of the O. We probably want to add a "free GPUs" or sth (notice that with the multi-stream per gpu implementation, we will see each slot as a GPU)